### PR TITLE
Correct reg URLs

### PR DESCRIPTION
**Pull request by gsvr**
Needs a API v2 config block, then reg use that with `/api/v2/registrations/` not `/api/v1/registration/`

Update: `control_api_call` with `Token: token`

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/utils.js [hunk 1]
```diff
@@ -611,7 +640,7 @@ go.utils = {
             authority: reg_type,  // 'clinic' | 'chw' | 'personal'
         };
         return go.utils
-            .control_api_call("post", null, payload, 'registration/', im)
+            .control_v2_api_call("post", null, payload, 'registration/', im)
```
Comment (imsickofmaps) [inline src/utils.js]: Should be `'registrations/'`

**Comment by imsickofmaps** [comment]
Also, I think it's wise to update the fixtures, even if headers aren't check at the moment

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**